src/Pages/Main/components/SubBanner.js

```diff
@@ -0,0 +1,75 @@
+import { useState } from "react";
+import styled from "styled-components";
+import leftBtn from "../images/left.png";
+import rightBtn from "../images/right.png";
+import banner01 from "../images/subBanner01.jpg";
+import banner02 from "../images/banner2.jpg";
+
+const SubBanner = () => {
+  const [trnaslateX, setTranslateX] = useState(0);
+  const slideHandler = (value) => {
+    if (trnaslateX + value > 0 || trnaslateX + value < -100) return;
+    setTranslateX(trnaslateX + value);
+  };
+  return (
```

Comment:
```suggestion
const SubBanner = () => {
  const [trnaslateX, setTranslateX] = useState(0);
  
  const slideHandler = (value) => {
    if (translateX + value > 0 || translateX + value < -100) return;
    setTranslateX(translateX + value);
  };
  
  return (
```
- 구분되는 로직 끼리는 한줄 띄워주세요
- translateX 오타입니다